src/openloco/localisation/FormatArguments.hpp

```diff
@@ -0,0 +1,41 @@
+#pragma once
+
+#include "../interop/interop.hpp"
+
+using namespace openloco::interop;
+
+namespace openloco
+{
+
+    class FormatArguments
+    {
+    private:
+        void* _buffer;
+        void* _bufferStart;
+
+    public:
+        FormatArguments(void* buffer)
+        {
+            _bufferStart = buffer;
+            _buffer = _bufferStart;
+        }
+
+        FormatArguments()
+            : FormatArguments(&addr<0x112C826, void*>())
```

Comment:
I'd like to see this named common format args. It would also be good if this wrapper had a max number of arguments before trashing memory.